src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -393,7 +393,10 @@ def get_deployment_layout(self) -> List["OSD"]:
         functions.
         '''
         devices = self.args.devices
-        fast_devices = self.args.db_devices
+        if self.args.block_db_size is not None:
+            fast_devices = self.args.db_devices or self.args.devices
```

Comment:
I can't really understand reading this why `fast_devices` has gained the ability to be either `args.db_devices` or `arg.devices` when it was always `args.db_devices` before

Reply:
From the commit message:

> Passing `--block-db-size` and `--db-devices` makes ceph-volume create db devices
on dedicated devices (current implementation)

> Passing `--block-db-size` without `--db-devices` makes ceph-volume create a separate
LV for db device on the same device